src/patternfly/components/Button/button.hbs

```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-button{{#if button--modifier}} {{button--modifier}}{{/if}}"
+<button class="pf-c-button{{#if button--modifier}} {{button--modifier}}{{/if}}" aria-hidden="true"
```

Comment:
Can you revert the change for this file so that the PR shows no changes to this file? Your previous edits are still showing up.

Reply:
file has been reverted back.